packages/start/src/server/handler.ts

```diff
@@ -31,18 +31,24 @@ export function getExpectedRedirectStatus(response: ResponseStub): number {
 export function createBaseHandler(
   fn: (context: PageEvent) => unknown,
   createPageEvent: (event: FetchEvent) => Promise<PageEvent>,
+  routerLoad?: (event: FetchEvent) => Promise<void>,
```

Comment:
this new argument, it's probably best to have it last, because it's the only optional, and then the order doesn't change for `options`

Reply:
good call not sure why exactly i put it there